modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portlet/_controls.scss

```diff
@@ -119,4 +119,8 @@ body.portlet {
 	.portlet-topper {
 		display: none !important;
 	}
+
+	.portlet-title-editable {
+		pointer-events: none;
```

Comment:
:thinking: This disables mouse events, but if people are using keyboard, can they still activate the item?

Reply:
@NemethNorbert :point_up: ?

Reply:
Looks like the portlet name is not under the tab order. Haven't thoroughly checked this but if we confirm that's the case, this solution would be enough. @NemethNorbert could you please confirm this?